src/base-command.ts

```diff
@@ -72,6 +72,9 @@ export default abstract class BaseCommand extends Command {
   }
 
   async finally(_: Error | undefined): Promise<any> {
+    if (_ instanceof ValidationErrors) {
+      process.exitCode = 1;
```

Comment:
Don't we want it to exit 1 for any error? Especially uncaught ones. 

Why are we calling `await this.endSentryTransaction();` in both finally and catch methods? I'd imagine it should only exist in the finally method since that will "should" always run.

Also why doesn't catch always call `super.catch(err);`? That would be the reason we aren't spitting out the correct error code.

Reply:
@mueschm you have more context and I'd def like to get a fix in for this tomorrow after the demo

Reply:
BaseCommand.catch attempts to send sentry metadata before calling the super.catch method in order to get running docker context written to sentry/file before any cleanup is performed.  

If the error is a validation error, or is err.oclif.exit === 0, it will not call super.catch from the catch method. I suppose this could be an artifact from when we were returning prettyValidationErrors(err) before sentry was added. 
If super.catch() is called with validation error, it will double print the error message to screen. I will re-evaluate the logic block.

Reply:
Will help to test this, but pretty sure we do just always want to return the 1 on this finally.